docs/wiki/testing/formal-verification.md

```diff
@@ -0,0 +1,172 @@
+# A brief introduction to formal verification
+
+## Mathematics and the art of abstraction
```

Comment:
```suggestion
## Intro to mathematical abstraction 
```

Or 'abstraction in math 101' or something like that might be a better title here. I know you like to intro with these cute stories but I would rather keep it a bit shorter and formal 😅 Consider making this section leaner with focus on the important pieces

Reply:
> I know you like to intro with these cute stories

![image](https://github.com/eth-protocol-fellows/protocol-studies/assets/10168946/25e1ad87-ce98-46c5-8f9a-995f335d7778)